game/build.gradle

```
@@ -0,0 +1,93 @@
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
+
+plugins {
+    id 'java-library'
+}
+
+dependencies {
+    // LibGDX
+
+    api supportDependencies.gdx
+    api supportDependencies.gdx_freetype
+    api supportDependencies.gdx_backend_lwjgl3
+    api supportDependencies.gdx_platform
+    api supportDependencies.gdx_freetype_platform
+    api supportDependencies.gdx_ai
+
+    // JUnit 4, Mockito and Powermock for testing:
+    // https://mvnrepository.com/artifact/junit/junit
+    testImplementation supportDependencies.junit
+    testImplementation supportDependencies.mockito_core
+
+    // https://mvnrepository.com/artifact/com.google.code.gson/gson
+    implementation supportDependencies.gson
+}
+
+// store name of method parameter names in .class files (used to build
+// DSL-adapter types, which contain the names of the parameters as members)
+tasks.withType(JavaCompile) {
+    configure(options) {
+        options.compilerArgs << '-parameters'
+    }
+}
+
+
+sourceSets.main.java.srcDirs = ["src/"]
```

```suggestion
sourceSets.main.java.srcDirs = ['src/']
```
hier auch, ' verwenden...